fix-process-detection.sh

```
@@ -0,0 +1,43 @@
+# =============================================================================
+
+# Función mejorada de detección de agente promiscuo
+check_promiscuous_agent_improved() {
+    # Buscar como usuario normal
+    if pgrep -f "promiscuous_agent" &>/dev/null; then
+        echo "✅ Agente Promiscuo encontrado (usuario normal)"
+        return 0
+    fi
+
+    # Buscar como root usando sudo
+    if sudo pgrep -f "promiscuous_agent" &>/dev/null 2>/dev/null; then
```

Shellcheck (0.10.0)

[error] Multiple redirections compete for stderr. Use cat, tee, or pass filenames instead.

(SC2261)